fab/inventory/guinea

```diff
@@ -0,0 +1,47 @@
+[proxy]
+guineact.commcarehq.org
+
+[webworkers]
+guineact.commcarehq.org
+
+[webworkers:vars]
+newrelic_app_name=django
+
+[couchdb]
+guineact.commcarehq.org
+
+[postgresql]
+guineact.commcarehq.org
+
+[rabbitmq]
+guineact.commcarehq.org
+
+[celery]
+# ansible makes the first thing in this list the flower url
+# a little bit redundant with with more explicit settings in environments.yml
+# but ansible doesn't currently have access to that
+guineact.commcarehq.org
+
+[celery:vars]
+guineact.commcarehq.org
+
+[pillowtop]
+guineact.commcarehq.org
+
+[touchforms]
+guineact.commcarehq.org
+
+[redis]
+guineact.commcarehq.org
+
+[memcached]
+guineact.commcarehq.org
+
+[elasticsearch]
+guineact.commcarehq.org
+
+[shared_dir_host]
```

Comment:
Just thinking about this again, I don't think Ansible is set up to have the host and the client as the same machine. I suspect it will try to create the shared drive and then share it to itself and mount it as an NFS drive which is obviously not necessary.

I'm not sure what the purpose of this guinea server is but possibly could just not use that feature. Alternatively just create a local folder on the server to use and don't treat it like an NFS drive.